src/backend/expungeservice/models/helpers/record_summarizer.py

```diff
@@ -0,0 +1,65 @@
+from expungeservice.models.expungement_result import ChargeEligibilityStatus
```

Comment:
```suggestion
from expungeservice.models.expungement_result.ChargeEligibilityStatus import *
```

So that `black` will allow you to keep it all inline. It currently looks unnecessarily spaced out. For example,

```
[c.expungement_result.charge_eligibility.status == ELIGIBLE_NOW for c in case.charges]
```

Reply:
oh, nice. Didn't know that was possible with enums